### ISSUE TITLE
signed versions

### ISSUE THREAD

**Issue by skaller** [calccrypto/uint256_t#12]
Any chance of signed versions? Should be easy enough, subtraction is just addition of two's complement, division and multiplication can be done by saving the sign, converting to unsigned, doing the operation, and converting back. In other words I think just a lightweight wrapper around the unsigned versions.

**Comment by calccrypto** [comment]
I intentionally wrote these as unsigned integers for usage in cryptography calculations. I did, however, write [integer](https://github.com/calccrypto/integer), which is an arbitrary precision signed integer type.

**Comment by skaller** [comment]
Yes, I've seen that too. Hope its ok, I'm trying to build uint256 as part of my system, then possibly integer. I can add int256 myself I suppose.
